### PR TITLE
Add colinslegacy.com instances

### PR DESCRIPTION
**Pull request by hendursaga**

### CODE REVIEW

File: services-full.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
       "https://reddit.smnz.de",
       "https://reddit.thekitty.zone",
       "https://safereddit.com",
-      "https://snoo.habedieeh.re"
+      "https://snoo.habedieeh.re",
+      "https://libreddit.colinslegacy.com"
```
Comment (benbusby) [inline services-full.json:67]: Libreddit instances are automatically updated nightly from https://github.com/libreddit/libreddit-instances/blob/master/instances.json

File: services-full.json [hunk 2]
```diff
@@ -99,7 +100,8 @@
       "https://teddit.tokhmi.xyz",
       "https://teddit.totaldarkness.net",
       "https://teddit.xbdm.fun",
-      "https://teddit.zaggy.nl"
+      "https://teddit.zaggy.nl",
+      "https://teddit.colinslegacy.com"
```
Comment (benbusby) [inline services-full.json:104]: Teddit instances are automatically updated from https://codeberg.org/teddit/teddit/raw/branch/main/instances.json

**Review by benbusby — CHANGES_REQUESTED** [review]
The piped and lingva instances are good to go, but the libreddit and teddit instances will need to be removed since they'll automatically get overwritten in the nightly build.

**Comment by hendursaga** [comment]
Oh, sorry, I forgot about that! Perhaps it would be good to have a short contributing guide?

**Comment by hendursaga** [comment]
(Sorry about that, I'm still trying to learn Git)

**Review by benbusby — APPROVED** [review]
No worries, thanks for updating!

**Pull request merged**